python/main.py

```diff
@@ -15,9 +18,17 @@ def main(use_existing_model=True):
     # Load data
     dataset_path = os.getenv('BEST_DATASET')
     df = pd.read_csv(dataset_path)
-    df_sampled_data = sample_flattened_dataset(df)
-    df_avg_data =  average_sampling(df)
-    X_train, X_test, Y_train, Y_test = get_train_test_split(df_avg_data)
+
+    # Colle filter. Slet når det er ændret i filerne
```

Comment:
Samme her. Det skriger også lidt på en lille abstraktion, når det her står der to gange, hvis det er tiden værd?